agent/config/runtime.go

```diff
@@ -1504,6 +1504,9 @@ type RuntimeConfig struct {
 	//
 	Watches []map[string]interface{}
 
+	// EnqueueLimit is used to specify a timeout value of Raft command apply
+	EnqueueLimit time.Duration
```

Comment:
Your comment says timeout and thats also the name for this variable in raft: https://github.com/hashicorp/raft/blob/e4251bbbec4d7ad67b394786de0c0d908d8d6253/api.go#L674. We have a couple of raft related configuration options and I would like to suggest another name: `RaftApplyTimeout`. Let me know if you have another idea.

I know this name existed before, but lets take that opportunity and make this piece of code a better place :).

Reply:
I see. I renamed the variable names to `RaftApplyTimeout`.